opensrp-app/src/main/java/org/smartregister/sync/helper/SyncSettingsServiceHelper.java

```diff
@@ -222,16 +222,4 @@ private JSONArray pullSettings(String directoryUrl, String accessToken) throws J
     protected Response<String> getResponse(String completeUrl, String accessToken) {
         return httpAgent.fetchWithCredentials(completeUrl, accessToken);
     }
-
-    public void setHttpAgent(HTTPAgent httpAgent) {
-        this.httpAgent = httpAgent;
-    }
-
-    public void setBaseUrl(String baseUrl) {
-        this.baseUrl = baseUrl;
-    }
-
-    public void setSharedPreferences(AllSharedPreferences sharedPreferences) {
-        this.sharedPreferences = sharedPreferences;
-    }
```

Comment:
Any reason you removed this?

Reply:
I had added the setters on another testing PR. Their role was to enable setting the values instead of using the constructor. I no longer need them

Reply:
cool